components/molecule/phoneValidation/demo/index.js

```diff
@@ -0,0 +1,47 @@
+import {useState} from 'react'
+
+import {AntDesignIcon} from '@s-ui/documentation-library'
+import AtomIcon from '@s-ui/react-atom-icon'
+
+import MoleculePhoneValidation from '../src/index.js'
+import {phoneValidationType} from '../src/settings.js'
+
+const prefixes = [
+  {value: 'ES', label: 'España', countryCode: '+34'},
+  {value: 'FR', label: 'Francia', countryCode: '+33'},
+  {value: 'PT', label: 'Portugal', countryCode: '+351'},
+  {value: 'AD', label: 'Andorra', countryCode: '+376'}
+]
+
+export const icon = (
+  <AtomIcon>
+    <AntDesignIcon icon={'AiOutlineDown'} style={{fill: 'currentColor'}} />
+  </AtomIcon>
+)
+
+export default () => {
+  const [phone, setPhone] = useState('')
+
+  return (
```

Comment:
I whould suggest you to see some other demo examples, it will help you to align the UI of the demo as is in other components demo

Compare the UI & styles from: 
- ✅  [This one](https://sui-components-r10onijy6-adevinta-spain.vercel.app/workbench/atom/button/demo)
- ❌ to [this one](https://sui-components-r10onijy6-adevinta-spain.vercel.app/workbench/molecule/phoneValidation/demo)

Take a look into Heading and block spacings